### PR TITLE
Add nutanix e2e network check

### PR DESCRIPTION
**Pull request by sgaoshang**
e2e test for case [OCP-51092 ](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-51092)- [ipi-on-nutanix] networking infrastructure components checking

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-disc-rt-f28

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-disc-rt-f28

**Comment by sgaoshang** [comment]
@jianlinliu pls help to review it, thanks.

File: ci-operator/step-registry/cucushift/installer/check/nutanix/network/cucushift-installer-check-nutanix-network-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ref:
+  as: cucushift-installer-check-nutanix-network
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/nutanix/network/cucushift-installer-check-nutanix-network-ref.yaml:2]: how about renaming it to `cucushift-installer-check-nutanix-infra`

Comment (sgaoshang) [reply ci-operator/step-registry/cucushift/installer/check/nutanix/network/cucushift-installer-check-nutanix-network-ref.yaml]: Fixed, thanks.

**Pull request merged**